Assets/Scripts/Item.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Item : MonoBehaviour {
+
+	public ItemSettings item;
```

Comment:
`public ItemSettings itemSettings { get; set; }`

if set is needed, else without set